tock/hours/forms.py

```diff
@@ -66,11 +69,14 @@ def render_option(self, selected_choices, option_value, option_label):
             if dict.get(option_label, 'notes_required'):
                 notes_required_html = ' data-notes-required="true"'
 
+            if dict.get(option_label, 'alerts'):
+                alerts_html = 'data-alerts="%s"' % (escapejs(json.dumps(option_label['alerts'])))
```

Comment:
I was not able to find a better alternative to turning the alert information into a JSON string and then escaping it on top of that in order to incorporate it into the existing page construction.  I'm open to suggestions if anyone is aware of a better way, though this does work.  However, it requires that `JSON.parse` be called on the value twice in the JavaScript side, which comes off as a code smell to me.